language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/util/definition/LSStandardLibCache.java

```diff
@@ -170,6 +172,27 @@ public Path getCachedStdlibRoot(String pkgName) throws LSStdlibCacheException {
         }
         return CommonUtil.LS_STDLIB_CACHE_DIR.resolve(pkgName);
     }
+    
+    private void populateLangLibs() {
+        List<String> langLibs = Arrays.asList("array", "decimal", "error", "float", "future", "int", "map", "object",
+                "stream", "string", "table", "typedesc", "value", "xml");
+        if (cacheUpdating) {
+            return;
+        }
+        new Thread(() -> {
+            try {
+                for (String langLib : langLibs) {
+                    String cacheableKey = "ballerina_lang." + langLib + "_0.0.0";
```

Comment:
Can we assume that the version will be `0.0.0`, for all the lang libs? or do we have a way to extract the bundled versions from the distribution itself, instead?

Reply:
@NipunaRanasinghe yeah. This is to avoid directory searching and pre load. But in future will have to find a dynamic way for doing this

Reply:
Ack :) +1